src/chrome/content/rules/Bilibili.com.xml

```diff
@@ -31,6 +33,7 @@
 		<test url="http://interface.bilibili.com/msg.xml" />
 	<target host="passport.bilibili.com" />
 	<target host="pay.bilibili.com" />
+	<target host="planet2017.bilibili.com" />
```

Comment:
XMLHttpRequest cannot load https://www.bilibili.com/activity/web/view/data/5. The 'Access-Control-Allow-Origin' header has a value 'http://planet2017.bilibili.com' that is not equal to the supplied origin. Origin 'https://planet2017.bilibili.com' is therefore not allowed access.